src/python/stup/utils.py

```diff
@@ -23,12 +25,9 @@
 def rand_str(length):
     return ''.join([
         random.choice(string.ascii_letters + string.digits)
-        for i in xrange(length)
+        for i in range(length)
```

Comment:
`xrange` is something conventional when you writing py2 code. 

i'll fix that later.